tests/cypress/tests-e2e/shareShortLink.cy.js

```diff
@@ -140,6 +140,12 @@ describe('Testing the share menu', () => {
                 `https://twitter.com/intent/tweet?text=&url=${encodeURIComponent(dummyShortLink)}`
             )
         })
+        it('opens a whatsapp popup with the short link as content', () => {
+            checkSocialNetworkType(
+                'whatsapp',
+                `https://api.whatsapp.com/send?text=%0D%0A${encodeURIComponent(dummyShortLink)}`
+            )
+        })
```

Comment:
Cypress `it()` are quite costly in test performance and cost (if we would have a non open source software it would cost), so now we tend to do less `it()`, I would suggest instead of adding a new `it()` (test case) to merge all in `it()` of the social network context into one `it()`. Each part of the test should be separate by a `cy.log()` statement to ease debugging (e.g. `cy.log('opens a whatsapp popup ....')`